fastestimator/cli/train.py

```diff
@@ -88,12 +91,13 @@ def configure_train_parser(subparsers: argparse._SubParsersAction) -> None:
                         dest='hyperparameters_json',
                         type=str,
                         help="The path to the hyperparameters JSON file")
-    parser.add_argument('--warmup', type=str, help="Warmup setting, can be true, false or debug", default="true")
+    parser.add_argument('--warmup', type=str, help="Warmup setting, can be true or false", default="true")
```

Comment:
we could switch this to boolean type rather than going thru string parsing

Reply:
smart